src/commands/rover/WhoisCommand.js

```diff
@@ -177,6 +206,62 @@ class WhoisCommand extends Command {
         if (Accolades[id]) embed.fields.push({ name: 'Accolades', value: `${Accolades[id]}`, inline: true })
 
         editMessage.edit({ embed: embed }).catch(console.error)
+        
+        let edited = false
+        let scriptingHelpers = ""
+        try {
+          const response = await request({
+            uri: `https://scriptinghelpers.org/resources/get_profile_by_roblox_id/${encodeURIComponent(data.robloxId)}`,
+            simple: false,
+            resolveWithFullResponse: true
+          })
+          scriptingHelpers = JSON.parse(response.body)
```

Comment:
You should check the status code here and if it 404s then don't attempt to decode the body

Reply:
Implemented in new commit.